server/src/main.rs

```diff
@@ -189,12 +189,40 @@ fn run_healthcheck(opts: RunOpts) -> Result<()> {
     std::process::exit(i32::from(failure))
 }
 
+async fn create_schema(database_url: String) -> Result<()> {
+    let sql_pool = {
+        let mut sql_opt = sea_orm::ConnectOptions::new(database_url.clone());
+        sql_opt
+            .max_connections(5)
```

Comment:
Nit: max_connections(1), we'll only ever need 1 connection.

Reply:
Still 5?

Reply:
oops, accidentally reverted the line when testing.